data/cities/detroit/corporations/dt-t.md

```diff
@@ -0,0 +1,19 @@
+name: DT&T
```

Comment:
Perso, je mettrais également mon véto à "Cheshire".
Pour ce qui est de "Mercury" et "EEC", je ne suis pas très convaincu, et j'ai un peu l'impression que personne ne l'est.
On devrait peut-être se laisser un peu plus de temps.